### PR TITLE
Update swagger-js with latest

### PR DESCRIPTION
**Pull request by whitlockjc**
* For the e2e tests to run, we had to enable CORS in http-server.  The
  release of http-server that supports CORS hasn't come out yet so we
  have to use http-server@master until then.  Here is the commit I speak
  of: https://github.com/nodeapps/http-server/commit/30f4d1e008b3d707a764eb06ad4c1ac2becd9e96

### CODE REVIEW

File: dist/swagger-ui.js [hunk 1]
```diff
@@ -783,9 +531,9 @@ this["Handlebars"]["templates"]["param_readonly"] = Handlebars.template({"1":fun
   if (stack1 != null) { buffer += stack1; }
   return buffer + "</td>\n<td><span class=\"model-signature\"></span></td>\n";
 },"useData":true});
-this["Handlebars"]["templates"]["param_readonly_required"] = Handlebars.template({"1":function(depth0,helpers,partials,data) {
+this["Handlebars"]["templates"]["param_readonly"] = Handlebars.template({"1":function(depth0,helpers,partials,data) {
```
Comment (mohsen1) [inline dist/swagger-ui.js]: @whitlockjc I think you need to run `gulp` for this branch again. There is no `.handlebars` file changed in this commit so this change looks suspicious

Comment (whitlockjc) [inline dist/swagger-ui.js]: I just ran `gulp` again and there are no updated files.  Here was my flow:

* Update the code
* Run `gulp`
* Run `npm test`

I committed after that.

**Comment by mohsen1** [comment]
@fehguy In order to fix bugs related to swagger-js we need to merge this

**Comment by fehguy** [comment]
but the e2e tests ran before, what has changed?

**Comment by mohsen1** [comment]
e2e tests passed with this. Code has changed since 2 days ago that this PR was made. @whitlockjc please rebase from develop_2.0 again. @fehguy Please merge it after rebase

**Comment by whitlockjc** [comment]
Done.

**Pull request merged**